### PR TITLE
CORTX-28697: use generic method variable names

### PR DESCRIPTION
**Pull request by rohit-k-dwivedi**
Signed-off-by: Rohit Dwivedi <rohit.k.dwivedi@seagate.com>

# Problem Statement
- Use generic names to avoid use of `rgw` 

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [x] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: src/rgw/setup/rgw.py [hunk 1]
```diff
@@ -157,7 +157,7 @@ def start(conf: MappedConf, index: str):
         log_path = Rgw._get_log_dir_path(conf)
         log_file = os.path.join(log_path, f'{COMPONENT_NAME}_startup.log')
         config_file = Rgw._get_rgw_config_path(conf)
-        RgwStart.start_rgw(conf, config_file, log_file, index)
+        RgwStart.start_service(conf, config_file, log_file, index)
```
Comment (cdeshmukh) [inline src/rgw/setup/rgw.py:160]: https://github.com/Seagate/cortx-rgw-integration/pull/54/files#r820413668

class & method name getting addressed in PR above

File: src/rgw/setup/rgw_service [hunk 2]
```diff
@@ -37,9 +37,9 @@ while [ $# -gt 0 ];  do
     shift 1
 done
 
-function start_rgw {
+function start_service {
```
Comment (cdeshmukh) [inline src/rgw/setup/rgw_service:40]: again, after PR #54 , this file wont be there. rebase after that is merged and then take this forward

File: cortx [hunk 3]
```diff
@@ -0,0 +1 @@
+/home/932781/PR/28697_remove_rgw_name/cortx-rgw-integration/src/
```
Comment (cdeshmukh) [inline cortx:1]: whats this ? remove from PR plz

Comment (rohit-k-dwivedi) [reply cortx]: done

**Comment by rahul-tripathi-git** [comment]
HI @cdeshmukh 
Just wanted to confirm that 'rgw' named variable names in build script and setup.py is fine right?
Or should they be also handled?

**Comment by rahul-tripathi-git** [comment]
Hi @rohit-k-dwivedi 
There are some conststants defined with *Rgw* names, please confirm if they should be left as is.

File: src/rgw/setup/rgw.py [hunk 4]
```diff
@@ -78,14 +78,14 @@ def prepare(conf: MappedConf):
...
-            Conf.save(Rgw._rgw_conf_idx)
-            Log.info(f'{RGW_CONF_TMPL} config copied to {rgw_config_path}')
+            config_path = Rgw._get_rgw_config_path(conf)
+            tmpl_idx = f'{COMPONENT_NAME}_conf_tmpl'  # e.g. rgw_conf_tmpl
+            tmpl_url = f'ini://{RGW_CONF_TMPL}'
+            Rgw._load_rgw_config(tmpl_idx, tmpl_url)
+            Rgw._load_rgw_config(Rgw._conf_idx, f'ini://{config_path}')
+            Conf.copy(tmpl_idx, Rgw._conf_idx)
+            Conf.save(Rgw._conf_idx)
+            Log.info(f'{RGW_CONF_TMPL} config copied to {config_path}')
```
Comment (cdeshmukh) [inline src/rgw/setup/rgw.py:88]: RGW_CONF_TMPL -> CONF_TEMPLATE

File: src/rgw/setup/rgw.py [hunk 5]
```diff
@@ -470,10 +470,10 @@ def _update_hax_endpoint(conf: MappedConf, data_pod_hostname: str):
         Log.info(f'Fetched motr_ha_endpoint from data pod. Endpoint: {motr_ha_endpoint}')
 
         rgw_config_path = Rgw._get_rgw_config_path(conf)
```
Comment (selvakumaar5496) [inline src/rgw/setup/rgw.py:472]: rgw_config_path --> config_path

**Review by cdeshmukh — COMMENTED** [review]
Remove RGW references from code comments, docstrings & logs also, where ever possible

**Review by sachitanands — APPROVED** [review]

File: src/rgw/setup/rgw_service (hunk 2 above)
Comment (rohit-k-dwivedi) [reply src/rgw/setup/rgw_service]: done

File: src/rgw/setup/rgw.py (hunk 4 above)
Comment (rohit-k-dwivedi) [reply src/rgw/setup/rgw.py]: done

File: src/rgw/setup/rgw.py (hunk 5 above)
Comment (rohit-k-dwivedi) [reply src/rgw/setup/rgw.py]: addressed

**Review by selvakumaar5496 — APPROVED** [review]

File: src/rgw/setup/rgw.py (hunk 4 above)
Comment (rohit-k-dwivedi) [reply src/rgw/setup/rgw.py]: done

**Review by cdeshmukh — APPROVED** [review]

**Pull request merged**